include/morphio/types.h

```diff
@@ -52,4 +60,22 @@ using MorphologyVersion = std::tuple<std::string, uint32_t, uint32_t>;
 template <typename T>
 using range = gsl::span<T>;
 
+
+struct CellFamily {
+    struct NEURON {
+        using Type = SectionType;
+        static constexpr int value = 0;
+    };
+    struct GLIA {
+        using Type = GlialSectionType;
+        static constexpr int value = 1;
+    };
+};
+
+using NeuronalSection = Node<CellFamily::NEURON>;
+using GlialSection = Node<CellFamily::GLIA>;
+
+// legacy name
+using Section = NeuronalSection;
```

Comment:
Note for the future. I think it would be best to stop using `Section` and to start using `NeuronalSection` when talking about neuronal sections.
I don't know how could we have a deprecation of `Section` in C++. Something like we would do in python:

```
"Section" is deprecated, please use "NeuronalSection" when refering to neuronal section
```